akp/resource_akp_cluster.go

```diff
@@ -275,6 +276,9 @@ func buildClusterApplyRequest(ctx context.Context, diagnostics *diag.Diagnostics
 func buildClusters(ctx context.Context, diagnostics *diag.Diagnostics, cluster *types.Cluster) []*structpb.Struct {
 	var cs []*structpb.Struct
 	apiCluster := cluster.ToClusterAPIModel(ctx, diagnostics)
+	if diagnostics.HasError() {
```

Comment:
what is the reason that we ignore the error here?

Reply:
@jiachengxu This doesn't ignore error but rather exits early when having errors, for example input validation error. And then the diagnostic errors will be reported to users.